src/ZombieDriver.php

```diff
@@ -94,13 +94,14 @@ public function stop()
     public function reset()
     {
         $js = <<<JS
-browser.destroy();
-browser = null;
```

Comment:
Well, you should write a test reproducing the actual workflow which caused issues for you (and this workflow cannot involve getting the text of the window just after resetting as this is not supported). This test would ensure that this workflow works fine on all drivers.